docs/src/autodiff.md

```diff
@@ -0,0 +1,5 @@
+# Gradient in AdvancedHMC.jl
+
+AdvancedHMC.jl supports automatic differentiation using [`LogDensityProblemsAD`](https://github.com/tpapp/LogDensityProblemsAD.jl) and user-specified gradients. While the default AD backend for AdvancedHMC.jl is ForwardDiff.jl, we can seamlessly change to other backend like Zygote.jl using various syntax like `Hamiltonian(metric, ℓπ, Zygote)`, `Hamiltonian(metric, ℓπ, Val(:Zygote))` or via ADTypes.jl `Hamiltonian(metric, ℓπ, AutoZygote())`.
```

Comment:
I wonder if we should use Mooncake as an example rather than Zygote, because Zygote is dying and we are trying to promote Mooncake as the replacement.